# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1734542683 -3600
# Parent  6af0c1f7342462af9e892e4fd2835b5396126fce
Branding: Don't offer exporting to 'Betterbird' Mobile.

diff --git a/mail/locales/en-US/messenger/preferences/preferences.ftl b/mail/locales/en-US/messenger/preferences/preferences.ftl
--- a/mail/locales/en-US/messenger/preferences/preferences.ftl
+++ b/mail/locales/en-US/messenger/preferences/preferences.ftl
@@ -1084,13 +1084,13 @@ sync-disconnected-text = Synchronize you
 sync-disconnected-turn-on-sync = Turn on Syncing…
 
 ## Mobile QR Export Pane
 
-qr-export-pane-header = Export accounts to { -brand-product-name } Mobile
+qr-export-pane-header = Export accounts to Thunderbird Mobile
 
 qr-export-description = Quickly transfer your account settings from desktop to mobile by generating a QR code. Select which accounts to include, decide if you want to transfer your password, and scan the code with your mobile device. Fast, secure, and simple.
 
-qr-export-get-app = Don’t have { -brand-product-name } on mobile yet? <a data-l10n-name="app-link">Get it on Google Play</a>
+qr-export-get-app = Don’t have Thunderbird on mobile yet? <a data-l10n-name="app-link">Get it on Google Play</a>
 
 qr-export-create = Create a QR code to export your accounts
 
 qr-export-select-accounts = Select which accounts to export:
